projects/plugins/jetpack/extensions/plugins/ai-assistant-plugin/components/ai-assistant-plugin-sidebar/index.tsx

```diff
@@ -69,13 +75,28 @@ export default function AiAssistantPluginSidebar() {
 			<JetpackPluginSidebar>
 				<PanelBody title={ title } initialOpen={ false }>
 					<PanelRow>
-						<Proofread busy={ isRedirecting } disabled={ requireUpgrade } />
+						<BaseControl
+							className="jetpack-ai-proofread-control__header"
+							label={ __( 'AI feedback on post', 'jetpack' ) }
+						>
+							<Proofread busy={ isRedirecting } disabled={ requireUpgrade } />
+						</BaseControl>
 					</PanelRow>
 					{ requireUpgrade && (
 						<PanelRow>
 							<Upgrade onClick={ autosaveAndRedirect } type={ upgradeType } />
 						</PanelRow>
 					) }
+					{ isUsagePanelAvailable && (
+						<PanelRow>
+							<BaseControl
```

Comment:
Are we sure we don't want the BaseControl inside UsagePanel? Seems to me the block could handle the entire component tree by itself.

Of course, this is no blocker and likely on your plans for later iterations :)

Reply:
I was going to add it inside the UsagePanel, but then I also needed to add a BaseControl around the Proofread component and was not thinking about changing it directly, so I added the BaseControl as a wrapper on both.

Thinking more about it, I think it's OK to add them on the component itself. I am starting to work on a follow-up now, so I am going to move the BaseControl inside the UsagePanel and the Proofread component.

Thank you for the comment 👍